src/themes/default/index.ts

```diff
@@ -0,0 +1,318 @@
+import accordion from "./accordion";
```

Comment:
Isn't this index file too big ? Maybe we can refactor into smaller files ?

Reply:
It was like this from before. This PR isn't addressing any refactor, it's simply adding typescript support